### PR TITLE
ci: Add `run-benchmark` ci

### PR DESCRIPTION
**Pull request by misonijnik**
- [x] Add initial version
- [x] Bootstrap only benchmarks running on ci
- [x] Bind a trigger to changes in files associated with benchmarks

### CODE REVIEW

File: bootstrap.sh [hunk 1]
```diff
@@ -37,51 +38,8 @@ fi
 
 cd $BASE_DIR
 
-SCRIPT_JAVA_HOME="/usr/lib/jvm/java-11-openjdk-amd64/"
-export JAVA_HOME="${SCRIPT_JAVA_HOME}"
+(cd $BASE_DIR; ./scripts/bootstrap-BenchmarkJava.sh $BOOTSTRAP_OPTIONS)
+(cd $BASE_DIR; ./scripts/bootstrap-BenchmarkJava-mutated.sh$BOOTSTRAP_OPTIONS)
+(cd $BASE_DIR; ./scripts/bootstrap-reality-check.sh$BOOTSTRAP_OPTIONS)
```
Comment (ocelaiwo) [inline bootstrap.sh:43]: spaces

File: bootstrap.sh [hunk 2]
```diff
@@ -10,23 +10,24 @@ requireCommand() {
...
 requireCommand docker
-requireCommand javac
-requireCommand mvn
 requireCommand python3
 
 SCRIPT_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" &> /dev/null && pwd )"
 BASE_DIR="$(cd $SCRIPT_DIR && pwd)"
 
 EXIT_ON_ERROR="false"
 UPDATE_BENCHMARKS="false"
```
Comment (ocelaiwo) [inline bootstrap.sh:19]: not needed anymore?

**Review by ocelaiwo — CHANGES_REQUESTED** [review]

File: bootstrap.sh (hunk 1 above)
Comment (misonijnik) [reply bootstrap.sh:43]: This is a trick. `BOOTSTRAP_OPTIONS` is empty by default, and its updates add a space and an option after, so the options are either `" or `" *options* *option* ... *option*". The additional space seems redundant, doesn't it?

**Review by ocelaiwo — CHANGES_REQUESTED** [review]

File: bootstrap.sh (hunk 1 above)
Comment (misonijnik) [reply bootstrap.sh:43]: I agree. It can become an issue eventually.

**Review by ocelaiwo — APPROVED** [review]

**Pull request merged**